core/arch/riscv/include/kernel/misc.h

```diff
@@ -0,0 +1,28 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
```

Comment:
This file should be removed from commit "riscv: include: add misc_arch.h". Once done that commit is:

`Reviewed-by: Jerome Forissier <jerome.forissier@linaro.org>`